cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -3,18 +3,25 @@ kind: StatefulSet
 metadata:
   labels:
     application: skipper-ingress-redis
-    version: v6.2.4
+    #
+    # skipper-ingress-redis podManagementPolicy update
+    # - version is updated with `-parallel` suffix to not match the version configured in deletions.yaml
+    # - `-parallel` suffix can be dropped after completion along with deletions.yaml cleanup
+    #
+    version: v6.2.4-parallel
```

Comment:
As far as I know the deletions would be applied before applying manifests so it should not be a problem ti have the same version.

Reply:
If this has the same version it will delete sts on every config change while we need to delete the old one (`version: v6.2.4`) only once